addons/web_unsplash/static/src/components/media_dialog/image_selector.js

```diff
@@ -146,9 +148,11 @@ patch(ImageSelector.prototype, 'image_selector_unsplash', {
 
     async loadMore(...args) {
         await this._super(...args);
-        const { records, isMaxed } = await this.fetchUnsplashRecords(this.state.unsplashRecords.length);
-        this.state.unsplashRecords.push(...records);
-        this.state.isMaxed = isMaxed;
+        return this.keepLastUnsplash.add(this.fetchUnsplashRecords(this.state.unsplashRecords.length)).then(({ records, isMaxed }) => {
```

Comment:
could remove the `()` around `{records, isMaxed}`, no?
And somehow could've split the line so it's no that long